src/modules/functions.js

```diff
@@ -0,0 +1,41 @@
+import taskList from './taskarray.js';
+import * as element from './elements.js';
+
+// Add Tasks to Array
+export const addTask = (x, y, z) => {
+  const obj = { a: x, b: y, c: z };
```

Comment:
- [ ] As requested by the previous reviewer, kindly replace these letters (a, x, b, y...) with descriptive variable names 👍